lib/gis/parser_dependencies.c

```diff
@@ -124,6 +124,8 @@ static int is_flag(const void *p)
     }
 
     G_fatal_error(_("Internal error: option or flag not found"));
+
+    return -1;
```

Comment:
I know this is under a G_fatal_error, so shouldn't be reached. But the 4 usages of this function in this file only check with `if (is_flag(p)) {}`, so a non-zero value, here -1, doesn't mean the same.

Reply:
Code after `G_fatal_error` is never reached and thus unnecessary. (Competent static analysers recognises this too).